lib/engine/ability/hex_bonus.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require_relative 'base'
+
+module Engine
+  module Ability
+    class HexBonus < Base
+      attr_reader :bonus_name, :hex, :bonus
```

Comment:
can just call it name

Reply:
maybe bonus would be better renamed to amount?

Reply:
Done